contracts/mocks/DestructibleDelayedMock.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.4.18;
+
+import "../lifecycle/DestructibleDelayed.sol";
+
+
+// mock class using BasicToken
```

Comment:
This comment doens't really apply here :stuck_out_tongue: Also, could you add a short comment mentioning that all the mock does is make the constructor `payable` to make testing easier?